cloud/shared/bin/doctor_impl

```diff
@@ -13,9 +13,6 @@ if not is_linux and not is_mac:
 class DockerCheck(Check):
     remedy = "Need to install Docker CLI: https://docs.docker.com/engine/install/"
 
-    def should_skip(self):
```

Comment:
Btw, is this accurate? I believe we use Docker no matter what.

Reply:
oof, definitely not accurate. ty!